sample-data/src/main/java/com/kodemon/sampledata/SampleDataLoadingFacadeImpl.java

```diff
@@ -65,16 +65,24 @@ public void loadData() throws PasswordStorage.CannotPerformOperationException {
         pikachu.setTrainer(ash);
         pokemonDao.save(pikachu);
 
+        Pokemon mewtwo = new Pokemon(PokemonName.MEWTWO);
+        mewtwo.setLevel(999);
+        pokemonDao.save(mewtwo);
+
         Trainer admin = new Trainer();
         admin.setUserName("admin");
         admin.setFirstName("Professor");
         admin.setLastName("Oak");
         Date dob2 = new Calendar.Builder().setDate(1963, 24, 3).build().getTime();
         admin.setDateOfBirth(dob2);
         admin.setAdmin(true);
+        admin.addPokemon(mewtwo);
 
         trainerService.register(admin, "adminpassword");
 
+        mewtwo.setTrainer(admin);
```

Comment:
I think that you should do this in reversed order. Firstly, save the pokemon and after that assign pokemon to the trainer. Because it will throw transientSomethingException.